Sponsors.md

```diff
@@ -1,90 +1,91 @@
 # Sponsors
 
-We thought long and hard about accepting sponsorships. You will find our thinking in the “History” section, at the end of this page. 
+## Main way to sponsor: OpenCollective
 
-_These days, it seems, everyone has some advertisement targeted to software developers..._
+The main way to sponsor DevStaff is through our OpenCollective page [here](https://opencollective.com/devstaff), which helps us gather financial contributions via a transparent system. There, you can find the available contribution options such as:
 
-Below you will find the available sponsorship packages / options: 
+* One-time contribution of a custom amount
+* One-time contributions of predefined amounts to sponsor a number of next meetups
 
-## DevStaff Swag Sponsor
-Help the community members wear the DevStaff community name with pride! With this option, you will sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
+As a thank you for sponsoring a number of next meetups, we will feature you in those meetups by mentioning you and your contribution.
 
-The swag pack will be awarded to lucky attendees through a lottery that will take place at the end of the meetup. Your company will be publicly thanked for making this possible. 
+### How financial contributions are used
 
-The swag pack will include: 
-* a dev-friendly cool looking t-shirt, 
-* “Raki” / “Tsikoudia” shot glasses! (we are the definitive developer community in Crete after all) 
-* (optionally) any other item that can fit in the budget
+The financial contributions may be used by DevStaff for various purposes to support the community, e.g., providing food at a meetup, ordering swag to give out to community members, etc. 
+These activities are expensed in accordance to OpenCollective's policies, including submitting a receipt for transparency purposes.
 
-Here’s how the swag draw will work: 
-* 5 swag packs will be awarded at each meetup
-* Each pack costs 20 EUR
-* 100 EUR buys us 5 packs - i.e. awareness for you for ONE meetup. 
-* 200 EUR buys us 10 packs - i.e. awareness for you for TWO meetups. 
-* 400 EUR buys us 20 packs - i.e. awareness for you for FOUR meetups. 
+## Other ways to sponsor
```

Comment:
I did not know if we want to remove these, so I left them, but removed specific amounts. I thought the amounts may be old and not up-to-date, so I just mention they should get into contact with us.